reconcile/gql_definitions/common/pagerduty_instances.gql

```diff
@@ -0,0 +1,9 @@
+# qenerate: plugin=pydantic_v1
+query PD {
```

Comment:
Query names must be unique globally, because all definitions share a common context. `PD` is very short and sth that might not be unique globally. I'd recommend to rely on more descriptive names here, like `PagerDutyInstances` or sth like that.

Reply:
I guess the reason this is short is because you'd like to have a short class name. I think a proper solution, without interfering with global gql context might be a feature flag in qenerate itself , like

```graphql
# qenerate: query_data_container_name=PD
query PagerDutyInstances { }
```

Reply:
The reason for this short query name is my laziness 😈 Let me change it